x/evm/types/msg.go

```diff
@@ -178,6 +179,59 @@ func (msg MsgEthereumTx) GetSignBytes() []byte {
 	panic("must use 'RLPSignBytes' with a chain ID to get the valid bytes to sign")
 }
 
+func ComputeSignBytes(chainID *big.Int, tx *ethtypes.Transaction) ([]byte, error) {
+	if chainID == nil {
+		txrlp, err := rlp.EncodeToBytes([]interface{}{tx.Nonce(), tx.GasPrice(), tx.Gas(), tx.To(), tx.Value(), tx.Data()})
```

Comment:
Undo, use `Signer.Hash(tx)` as before

Reply:
then, all unit test breaks

Reply:
also previous version, Hashed signature went to "Sign", bypasssed `Sign` because it's 32 length.

Reply:
I see, maybe we shouldn't hash the during `privkey.Sign()` then?

Reply:
yes, i think so